draft-ietf-dnsop-svcb-https.md

```diff
@@ -1589,6 +1589,25 @@ two `list-value`s:
     part1
     part2,part3
 
+# HTTP Mapping Summary
+
+This table serves as a non-normative summary of the HTTP mapping for SVCB.
+Future protocol mappings may provide a similar summary table.
+
+|                                  |                                          |
+|----------------------------------+------------------------------------------|
+| **Mapped scheme**                | "https"                                  |
+| **RR type**                      | HTTPS                                    |
+| **Name prefixing**               | None for port 443, otherwise Port-Prefix |
```

Comment:
Include a reference for "Port-Prefix"?  (and/or give an example?)

Reply:
I changed this to an example/template.